packages/dashboard/src/customization/widgets/lineScatterChart/component.tsx

```diff
@@ -200,8 +203,7 @@ const LineScatterChartWidgetComponent: React.FC<LineScatterChartWidget> = (widge
         thresholds={thresholds}
         significantDigits={significantDigits}
         size={size}
-        legend={legend?.visible == true ? {} : undefined}
-        defaultVisualizationType={mapConnectionStyleToVisualizationType(line?.connectionStyle)}
+        legend={legend?.visible == true ? emptyLegendObject : undefined}
```

Comment:
almost never use == , 

also, the legend prop being an object may not make sense, AFAIK, we almost never use those, probably worth just make it a boolean

Reply:
the type of legend is an object in line scatter chart properties. It's used as such in convertLegend so keeping it as a object. 

-- I did try making into a boolean and ran into "Property 'position' does not exist on type 'boolean'" on a few files.